### PR TITLE
Add CreateClusterForm

### PR DESCRIPTION
**Pull request by jtomasek**
Introduce basic form fields handling
Add @patternfly/react-icons to allow for using icons

### CODE REVIEW

**Comment by honza** [comment]
This works:

```tsx
const RedHatAccountFields: FC<RedHatAccountFieldsProps> = (
  props: RedHatAccountFieldsProps
): JSX.Element => {
  const {
    onProvidePullSecret,
    handleUsernameChange,
    handlePasswordChange,
    username,
    password
  } = props;
  return (
    <Fragment>
      <TextContent>
        
```

**Comment by jtomasek** [comment]
Fixed, thanks!

File: src/components/createCluster/PullSecretFields.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React, { FC, Fragment } from 'react';
+import { FormGroup, TextArea, TextContent, Text } from '@patternfly/react-core';
+
+import ExternalLink from '../ui/ExternalLink';
+
+interface PullSecretFieldsProps {
+  onProvideCredentials(): void;
```
Comment (honza) [inline src/components/createCluster/PullSecretFields.tsx]: Nit: Better written as 

```tsx
interface PullSecretFieldsProps {
  onProvideCredentials: () => void;
  handlePullSecretChange: (value: string) => void;
  pullSecret: string;
}
```

File: src/components/createCluster/RedHatAccountFields.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import {
+  FormGroup,
+  TextInput,
+  TextContent,
+  Text
+} from '@patternfly/react-core';
+
+import ExternalLink from '../ui/ExternalLink';
+
+interface RedHatAccountFieldsProps {
```
Comment (honza) [inline src/components/createCluster/RedHatAccountFields.tsx]: Nit: better written as

```tsx
interface RedHatAccountFieldsProps {
  onProvidePullSecret: () => void;
  handleUsernameChange: (value: string) => void;
  handlePasswordChange: (value: string) => void;
  username: string;
  password: string;
}
```

File: src/components/ClusterWizard.tsx [hunk 3]
```diff
@@ -3,14 +3,16 @@ import React, { FC, Fragment } from 'react';
...
+import CreateClusterForm from './createCluster/CreateClusterForm';
 
 const ClusterWizard: FC = (): JSX.Element => (
   <Fragment>
     {/* <ClusterWizardSteps
       steps={[{ title: 'First' }, { title: 'Second' }]}
       currentStepIndex={0}
     /> */}
-    <BaremetalInventory />
+    {/* <BaremetalInventory /> */}
```
Comment (honza) [inline src/components/ClusterWizard.tsx]: This leaves the import as unused.

**Pull request merged**